apm-agent-core/src/main/java/co/elastic/apm/agent/impl/ElasticApmTracer.java

```diff
@@ -590,7 +590,9 @@ public synchronized void stop() {
         } catch (Exception e) {
             logger.warn("Suppressed exception while calling stop()", e);
         }
-        LoggingConfiguration.shutdown();
+        if (!assertionsEnabled) {
```

Comment:
[minor] maybe add a comment to explain why we do this when assertions are enabled so one of our future selves would not wonder why we do this.